### PR TITLE
Profile Page member

### LINKED ISSUE #1085: [NG] Improvise the Community Members Page on layer5-ng
**Current Behavior**
The current [community members](https://layer5ng.netlify.app/community/members) page looks a bit plain and simple.

**Desired Behavior**
Try out some different ways of representing each member card on the page.
Also, incorporate 4 member cards in a single row as compared to 3 member cards currently.
Incorporate LinkedIn profile links along with github and twitter in member card.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by vinayaksh42**
Signed-off-by: Vinayak Sharma <vinayaksh42@gmail.com>

**Description**
Profile page improvements

This PR fixes #1085 

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Member-single/index.js [hunk 1]
```diff
@@ -23,24 +23,24 @@ const MemberSingle = ({frontmatter}) =>{
...
                                 }
 
                                 {
                                     bio &&
-                                    <h6>{bio}</h6>
+                                    <h4>{bio}</h4>
                                 }
                                 {
                                     skills &&
                                     <h4>skills: {skills}</h4>
```
Comment (Tanuj22) [inline src/sections/Member-single/index.js:39]:
```suggestion
                                    <h4>Skills: {skills}</h4>
```

Comment (Tanuj22) [reply src/sections/Member-single/index.js:39]: Also make `Skills` bold

File: src/sections/Member-single/index.js [hunk 2]
```diff
@@ -23,24 +23,24 @@ const MemberSingle = ({frontmatter}) =>{
...
+                            <div className="body">
                                 {
                                     location &&
                                     <h4>Location: {location}</h4>
                                 }
 
                                 {
                                     bio &&
-                                    <h6>{bio}</h6>
+                                    <h4>{bio}</h4>
```
Comment (Tanuj22) [inline src/sections/Member-single/index.js:35]: Can you avoid changing the font-weight of the header tags (also do other headers tags)? You can create a class and use that in a `div` or a `span`

File: src/sections/Member-single/index.js [hunk 3]
```diff
@@ -23,24 +23,24 @@ const MemberSingle = ({frontmatter}) =>{
                                 { position && (
                                     <h2>{position}</h2>
                                 )}
+                            </header>
+                            <div className="body">
                                 {
                                     location &&
                                     <h4>Location: {location}</h4>
```
Comment (Tanuj22) [inline src/sections/Member-single/index.js:30]: Make `Location` bold

**Review by Tanuj22 — COMMENTED** [review]
Looks good. Some minor changes. There is unnecessary margin both at the top and bottom in mobile view. Remove that

**Comment by Nikhil-Ladha** [comment]
The design is definitely an improvement here :+1: 
One thing to change here is:
- On the full page, please remove the green color on the GitHub and other icons and keep them in grayscale and fill them on hover.

**Comment by Nikhil-Ladha** [comment]
These look better:
Screenshot from 2020-11-20 18-30-49
Screenshot from 2020-11-20 18-30-51

File: src/sections/Member-single/index.js (hunk 2 above)
Comment (Tanuj22) [reply src/sections/Member-single/index.js:35]: This is left

**Review by akashgp09 — COMMENTED** [review]
Looks better Now. Nice work @vinayaksh42

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
The green bg color on the hover is still there. Can you please remove that?

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**